api/webhook-vs-polling.md

```diff
@@ -93,20 +92,20 @@ Con's:
 
 * Not real time, a polling period is typically 5 mins.
 * You need to build or configure a periodic polling pattern at your side.
-* You cannot filter on which order events to receive, you will receive any order line change.
-* You cannot see what order event happened.
+* You cannot filter on which order or shipments events to receive, you will receive any order line change.
```

Comment:
```suggestion
* You cannot filter on which order or shipments events to act on, you will receive any order line / shipment line change.
```

Reply:
Used `any order or shipment change` as lines are not relevant.